### ISSUE TITLE
Content will be removed when adding an pagepart

### ISSUE THREAD

**Issue by dannyvw** [Kunstmaan/KunstmaanBundlesCMS#1776]
When adding a pagepart containing an editor (not in an nested pagepart) and you fill it with content and  add the same pagepart again then your content will be removed from all added pageparts. This does not happen when you save the node in the meantime. 

It looks like there is something wrong within the reinit https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/master/src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js#L105
  
Any ideas?

**Comment by Devolicious** [comment]
@dannyvw, So for a practical example it would be adding a TextPagePart, fill it wit some content, then add another TextPagePart and the content of the first will be removed?

**Comment by dannyvw** [comment]
Yes, but it does not seem to happen on http://demo.bundles.kunstmaan.be/en/admin. So maybe it's only in older versions?

**Comment by Numkil** [comment]
@dannyvw I tried testing this with a clean copy of 5.0 branch and 4.1 branch. both did not have the issue. Can you go to the following url in your project `/admin/settings/bundle-version` and give me the commit hash that is being the shown on that page?

**Comment by dannyvw** [comment]
@Numkil We are currently using version 3.5 and will soon update to the latest version.

**Comment by Numkil** [comment]
@dannyvw so I did a clean install of the oldest release that was easily accessible to me on github which was 3.6 and the issue did not occur there either. The file _pagepart-editor.js has also been changed extensively so there is a large chance that the bug you are experiencing has long been removed. 

I am going to close this ticket but if you notice that after you have upgraded your project the problem still persists, do not hesitate to reopen the ticket. And I will do my best to help you further.
